app/views/vzory/prieskum-spokojnosti.njk

```diff
@@ -0,0 +1,91 @@
+{% extends "layout.njk" %}
+{% from "breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "../../../src/idsk/components/header-extended/macro.njk" import idskHeaderExtended %}
+{% from "../../../src/idsk/components/customer-surveys/macro.njk" import idskCustomerSurveys %}
+
+{% set pageName = "Prieskum Spokojnosti" %}
+
+{% block beforeContent %}
+  <div class="govuk-width-container">
+    {{ govukBreadcrumbs({
+    "items": [
+    { text: ' ID-SK frontend', href: '/' + legacyQuery },
+    { text: pageName }
+    ]
+  }) }}
+  {% endblock %}
+
+  {% block content %}
+    <div style="margin-top: -20px; margin-bottom: -30px;">
```

Comment:
inline style nepouzivaj.. malo by sa to dat spravit nejako takto:
{% block head %}
<style>
.xzy {
// css style here
}
</style>
{% endblock %}

treba ceknut ale jak sa vola ten block nesom si isty presne